lib/shared/remove-package.ts

```diff
@@ -0,0 +1,54 @@
+import { getPackageManager } from "./get-package-manager"
+import { normalizePackageNameToNpm } from "./add-package"
+import path from "node:path"
+import fs from "node:fs"
+/**
+ * Removes a tscircuit component package.
+ * Handles different package name formats and uses the appropriate package manager.
+ *
+ * @param componentPath - The component identifier (e.g., author/name, @tsci/author.name)
+ * @param projectDir - The root directory of the project (defaults to process.cwd())
+ */
+export async function removePackage(
+  componentPath: string,
+  projectDir: string = process.cwd(),
+) {
+  const packageName = normalizePackageNameToNpm(componentPath)
+
+  console.log(`Removing ${packageName}...`)
+
+  const isTestMode = process.env.TSCI_TEST_MODE === "true"
```

Comment:
```suggestion
  const isTestMode = process.env.NODE_ENV=== "test"
```

is something like that working?